packages/job-worker/src/playout/datastore.ts

```diff
@@ -0,0 +1,23 @@
+import { deserializeTimelineBlob } from '@sofie-automation/corelib/dist/dataModel/Timeline'
+import { JobContext } from '../jobs'
+import { CacheForPlayout } from './cache'
+
+export async function cleanTimelineDatastore(context: JobContext, cache: CacheForPlayout): Promise<void> {
```

Comment:
This function could use a comment describing what it does.
"Remove entries from TimelineDatastore that have no references left in the Timeline"